man/qmethod.Rd

```diff
@@ -1,18 +1,40 @@
 \name{qmethod}
 \alias{qmethod}
 \title{Q methodology analysis}
-\description{This function performs a full Q method analysis using principal components analysis (see Notes). The main results are factor characteristics, statement z-scores and factor scores, and distinguising and consensus statements.}
+\description{This function performs a full Q method analysis using principal components analysis (see Notes). The main results are factor characteristics, statement z-scores and factor scores, and distinguishing and consensus statements.}
 \usage{
-qmethod(dataset, nfactors, rotation = "varimax", forced = TRUE, 
-        distribution = NULL, cor.method = "pearson", ...)}
+  qmethod(
+    dataset,
+    nfactors,
+    rotation = "varimax",
+    forced = TRUE,
+    distribution = NULL,
+    cor.method = "pearson",
+    reorder = FALSE,
+    quietly = FALSE,
+    ...
+  )
+}
 
 \arguments{
   \item{dataset}{a matrix or a data frame containing original data, with statements as rows, Q-sorts as columns, and the column scores in the distribution in each cell. The matrix or data frame should not contain character strings. The results keep row names and column names if set in the \code{dataset} (see 'Details').}
-  \item{nfactors}{number of factors to extract.}
+  \item{nfactors}{
+    number of factors to extract.
+    Consider the \code{\link{q.nfactors}} function for further advice on how many factors to extract.
+  }
   \item{rotation}{rotation method, defaults to \code{"varimax"}. Other possible rotations allowed in \code{\link[psych]{principal}} function can be used: \code{"none"}, \code{"varimax"}, \code{"quartimax"}, \code{"promax"}, \code{"oblimin"}, \code{"simplimax"}, and \code{"cluster"}.}
   \item{forced}{logical; Is the ranking of the items forced to match the distributions? Set to \code{TRUE} if all respondents ranked the items strictly following the distribution scores, in which case the values of the distribution are calculated automatically. Set to \code{FALSE} if respondents were able to rank the items without following the distribution, and the values of the distribution have to be provided as an array in the argument \code{distribution}.}
 \item{distribution}{logical; when forced = \code{FALSE}, the distribution has to be provided as a vector of numbers, such as \code{c(-2, -1, -1, 0, 1, 1, 2, 2)}.}
 \item{cor.method}{character string indicating which correlation coefficient is to be computed, to be passed on to the function \code{\link[stats]{cor}}: \code{"pearson"} (default), \code{"kendall"}, or \code{"spearman"}. }
+\item{reorder}{
+  Logical, indicating whether rotated components should be returned in decreasing order of the sums of their squared loadings.
+  Defaults to \code{FALSE} for compatibility with \code{\link{q.mrot.do}} and \code{\link{q.mrot.choose}}.
```

Comment:
Put default to TRUE, as it was before. In the instructions for manual rotation, you can indicate that this must be changed